test/upload/signature_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+ describe 'upload-signature' do
+   it 'returns 400 with nothing to sign' do
+     resp = Excon.get(CONFIG.UPLOAD_HOST)
+     resp.status.must_equal 400
+   end
+
+   it 'return plain text media type' do
+     connection = Excon.new(CONFIG.UPLOAD_HOST)
+     resp = connection.get(query: {to_sign: 'hello'})
+     resp.headers['content-type'].must_equal 'text/plain'
+   end
+
+   it 'return to correct signature' do
+     connection = Excon.new(CONFIG.UPLOAD_HOST)
+     resp = connection.get(query: {to_sign: 'test'})
```

Comment:
@gcampo88 This should be hitting /sign, right? (which can't be in UPLOAD_HOST, as we are now only using those for FQD's)